src/tasks/fetcher.py

```diff
@@ -89,6 +91,8 @@ async def fetch(self):
                         for team in teams_by_id.values()
                     ]
 
+                    self.storeman.storeScores(counts)
+
```

Comment:
I'd make this async as well. If for some reason the disk is slow, we don't wait for it. This should also be done for settings I notice now.

Would be ok for me to create a separate issue for this :D

Reply:
I've fixed it for the storeman, open a issue for settings as it is a bit out of scope for this PR :sweat_smile:

Reply:
Tis doesn"t fix it however, just wraps it in an extra layer of async :D. async does cooperative multiprocessing, it will not do anything else wile the file operations using `open` etc are in progress. It might still get stuck at some point. But still merging for now because the odds of this happening are low :D